forc-pkg/src/pkg.rs

```diff
@@ -1640,59 +1638,11 @@ pub fn dependency_namespace(
             }
         };
         root_namespace.add_external(dep_name, dep_namespace);
-        let dep = &graph[dep_node];
-        if dep.name == CORE {
-            core_added = true;
-        }
-    }
-
-    // Add `core` if not already added.
-    if !core_added {
```

Comment:
Shouldn´t this method do the same thing, but for the `std` lib?

Reply:
does the implicitly_include_std_if_missing function not do it?